lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex

```diff
@@ -166,6 +167,7 @@ defmodule LambdaEthereumConsensus.ForkChoice do
 
     Handlers.notify_forkchoice_update(store, head_block)
 
+    OperationsCollector.notify_new_block(head_block)
```

Comment:
nice! Seems this `recompute_head` has grown its responsabilities, basically doing every task that is async. We may want to do a refactor soon (out of scope of this PR)

Reply:
👍 #925